joint_names: [lwr_0_joint, lwr_1_joint, lwr_2_joint, lwr_3_joint, lwr_4_joint, lwr_5_joint, lwr_6_joint]

#variance in [rad]  don't change value (best tuning in simulation)
pos_meas_noise_var: 0.0000001 #0.001
process_noise_var: [0.0001, 0.0001, 2000.0] #[0.0, 0.0, 10.0]
prior_var: [0.0001, 0.0001, 100.0] #[0.0001, 0.5, 0.5]

loop_frequency: 50

delta_t: 0.02


############################################################################
#                                                                          #
#   FOR WILL OF GOD   DON'T CHANGE !!!!                                    #
#   pos_meas_noise_var: 0.0000001                                          #
#   process_noise_var: [0.0001, 0.0001, 2000.0]                            #
#   prior_var: [0.0001, 0.0001, 100.0]                                     #
#                                                                          #
#   loop_frequency: 50                                                     #
#                                                                          #
#   delta_t: 0.02                                                          #
#                                                                          #
############################################################################